src/actions/tamanu/psql.rs

```diff
@@ -0,0 +1,112 @@
+use std::fs;
+
+use clap::Parser;
+use miette::{miette, Context as _, IntoDiagnostic, Result};
+use tracing::{debug, instrument};
+
+use crate::actions::Context;
+
+use super::config::{merge_json, package_config};
+use super::{find_tamanu, TamanuArgs};
+
+/// Connect to Tamanu's db via `psql`.
+#[derive(Debug, Clone, Parser)]
+pub struct PsqlArgs {
+	/// Package to look at
+	#[arg(short, long)]
+	pub package: String,
```

Comment:
I think it would be better to try both packages automatically and go with the first config found, that way there's no need to think when calling the command.